Bozidar_Yukopila/Tarea-1/index.css

```diff
@@ -0,0 +1,173 @@
+
+*{
+    margin: 0;
+    padding: 0;
+   
+}
+  
+  
+@font-face {
+    font-family: tipohalo;
+    src: url(halo.ttf);
+}
+
+
+.fondo {
+    display: block;
+    margin: 0px auto;
+    width: 1000px;  
+    height: 1350px;
+    
+}
+
+.titulo {
+  display: flex;
+  justify-content: center;
+  align-items: center ;
+  position: relative;
+  top : -1300px ; 
+}
+.logo{
+    border-radius:50% ;
+    position: relative;
+    left:-40px ;
+    width: 250px;  
+    height: 250px;
+}
+.XBOX{
+   position: relative;
+   top: -50px; 
+   color : green ; 
+   font-size: 12em;
+}
+.mensaje{
+    text-align: center ;
+    
+    position: relative;
+    top: -1400px; 
+    left: 150px;
+    color :black ; 
+    font-size: 2.5em;
+ }
+
+ .HALO{
+    text-align: center ;
+    font-family: tipohalo;
+    color: rgb(6, 197, 204) ; 
+    font-size: 10em;
+    position: relative;
+    top: -800px; 
+    
+   
+ }
+ .INFINITE{
+    text-align: center ;
+    color: rgb(255, 255, 255) ;
+    font-size: 10em;
+    letter-spacing: 0.1em;
+    font-weight: 10;
+    font-family:  Arial;
+    position: relative;
+    top: -850px; 
+ }
+ .extra{
+
+    margin: 0 auto;
+    position: relative;
+    top: -1550px; 
+    left: 300px;
+    width: 300px;  
+    height: 300px;
+    background-color: green;
+    border-radius:50% ;
+ }
+ 
+ .console{
+
+    position: relative;
+    width: 250px;  
+    height: 200px;
+    top: 20px; 
+    left: 15px;
+    
+ }
+ .text_capsula{
+    width: 200px;  
+    height: 100px;
+ }
+
+ .texto{
+    text-align: center ;
+    font-size: 1em;
+    position: relative;
+    top : 10px ; 
+    left: 50px;
+    font-weight: 900 ; 
+    font-family: Helvetica ; 
+    color: white; 
+ }
+
+ .info{
+    margin: 0 auto;
+    position: relative;
+    top: -1850px; 
+    left: -250px;
+    width: 400px;  
+    height: 300px;
+   
+    
+ }
+
+ .texto2{
+
```

Comment:
@Bozidary24 , i see there are a few format problems , procure to use formatter like. prettier for VS Code